frontend/locales/ca.json

```diff
@@ -165,6 +165,7 @@
         "next_page": "Pàgina següent",
         "no_results": "No s'ha trobat cap element",
         "notes": "Notes",
+        "only_without_photo": "Només articles sense foto",
```

Comment:
How are these being translated? Machine based translations?

If they are machine based translations, we ask that only the English translations, and other languages you natively speak be included in the PR. And our community can handle translating this string to all the other supported languages.

Reply:
I used a translator service for those yes, but if you prefer I can remove them and only leave the English ones for now.